introduction.md

```diff
@@ -1 +1,5 @@
 Hello my name is jacekelgda and I like JavaScript path on codeschool.
```

Comment:
Since you're becoming a GitHub master, could you also add "Killing history using git rebase" as one of your deadly skills?